### ISSUE TITLE
Error - Invalid window id when using :help

### ISSUE THREAD

**Issue by wulymammoth** [shortcuts/no-neck-pain.nvim#128]
## Description

Getting an error when using `:h` or `:help`

```
Error executing vim.schedule lua callback: ...nvim/plugged/no-neck-pain.nvim/lua/no-neck-pain/main.lua:134: Invalid window id: 1020
stack traceback:
        [C]: in function 'nvim_win_get_width'
        ...nvim/plugged/no-neck-pain.nvim/lua/no-neck-pain/main.lua:134: in function <...nvim/plugged/no-neck-pain.nvim/lua/no-neck-pain/main.lua:106>
```

## Steps to reproduce

invoked `:h splitright`

## Expected behavior

No error

## Environment

- Neovim version: [e.g. 0.5.x / 0.6.x / 0.7.x / 0.8.x / Nightly]
- no-neck-pain.nvim version: [e.g. latest / 0.1.2 / dev]
- Plugin clash: [e.g. Telescope float window / lsp diagnostic]

**Comment by shortcuts** [comment]
hey there, thanks for opening the issue! I'm not able to reproduce the issue, but can add a safety check just in case

**Comment by shortcuts** [comment]
released in 0.3.1! let me know if it fixes it correctly :)
